### PR TITLE
feat: add descriptions to just commands

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

File: usr/share/ublue-os/just/custom.just [hunk 1]
```diff
@@ -122,35 +131,37 @@ jetbrains-toolbox:
...
 
+# Install nix and Devbox (Global Profile)
 nix-devbox-global:
   echo 'Installing devbox global profile.'
   devbox global pull https://devbox.getfleek.dev/high
   echo 'run "devbox global run install-bash-hook" to configure bash shell'
   echo 'run "devbox global run install-zsh-hook" to configure zsh shell'
   echo 'run "devbox global run" to see other available configuration commands'
 
-tea:
```
Comment (bobslept) [inline usr/share/ublue-os/just/custom.just:139]: Was this intended?

**Review by bobslept — COMMENTED** [review]
Very nice! Left a small note on the `tea` command to be sure.

File: usr/share/ublue-os/just/custom.just (hunk 1 above)
Comment (castrojo) [reply usr/share/ublue-os/just/custom.just:139]: Yeah I tried it for a bit but feels like we have more established things in here. We could leave it, just feels like it's not really picking up steam?

Comment (bobslept) [reply usr/share/ublue-os/just/custom.just:139]: Let's clean it up then

**Comment by castrojo** [comment]
Woo! Got them in config as well, we should be nice and descriptive across the board! https://github.com/ublue-os/config/pull/94

**Review by bsherman — APPROVED** [review]
Looks good to me :-)

**Pull request merged**